components/bookmarksList/index.jsx

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React, { useContext, useEffect, useState } from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import { createStyles, TextField } from "@material-ui/core";
+import Skeleton from "@material-ui/lab/Skeleton";
+import { makeStyles } from "@material-ui/styles";
+import { useBem } from "@solid/lit-prism-patterns";
+import {
+  Container,
+  PageHeader,
+  Table as PrismTable,
+} from "@inrupt/prism-react-components";
+import { Table, TableColumn } from "@inrupt/solid-ui-react";
+import { getThingAll } from "@inrupt/solid-client";
+import { dct } from "rdf-namespaces";
+import BookmarksContext from "../../src/contexts/bookmarksContext";
+import Bookmark from "../bookmark";
+import ResourceLink from "../resourceLink";
+import SortedTableCarat from "../sortedTableCarat";
+import Spinner from "../spinner";
+import styles from "./styles";
+import { RECALLS_PROPERTY_IRI } from "../../src/solidClientHelpers/bookmarks";
+import useResourceOwner from "../../src/hooks/useResourceOwner";
+
+const useStyles = makeStyles((theme) => createStyles(styles(theme)));
+
+function OwnerDisplayName({ iri }) {
```

Comment:
Might be overkill, but should these be in their separate files? E.g. `components/bookmarksList/ownerDisplayName/index.jsx`.

Reply:
I do like separate files, could go either way on this small component.